added the check for response processed, and renamed success to send

Rutger798 opened the pull request.

Commit:
added the check for response processed, and renamed success to send

ava57r commented on src/lib.rs:
```diff
@@ -168,8 +168,10 @@ pub struct Response {
         self.response == "success"
     }
+    /// Verifies if the request is successful processed
+    pub fn is_processed(&self) -> bool { self.info.contains("processed: 1") }
```
Maybe we should do an analysis of all the values, for example [code](https://github.com/andreevlex/zabbix-sender-cs/blame/master/ZabbixSender/SenderResponse.cs#L50), as did for us C#
add
- Processed
- Failed
- Total
- SecondsSpent

ava57r commented on src/lib.rs:
```diff
@@ -168,8 +168,10 @@ pub struct Response {
-    /// Verifies successful if there was a request
-    pub fn success(&self) -> bool {
+    /// Verifies if the date successful send
+    pub fn is_sent(&self) -> bool {
```
breaking change. Is it necessary to rename?

Commit:
Impl for handling the reset of info response

Rutger798 commented:
You have a valid point, so i have reverted the name change, and implemented the other status like you did in C#.

I'am bit new to rust and very new to github so just let me know if a should do something different.

ava57r commented:
Thanks

Rutger798 commented:
Thank you for merging!
